### PR TITLE
Triage usability improvements

### PR DESCRIPTION
**Pull request by rmmh**
- Better sorting for jobs and builds.
- You can now link to a specific cluster, and not have to view an overwhelming number of *other* clusters.
- There is a tooltip describing what the sparklines mean.

[staging](https://storage.googleapis.com/k8s-gubernator/triage/staging/index.html), [single-cluster link](https://storage.googleapis.com/k8s-gubernator/triage/staging/index.html#eba1ba01921bf541729e)

Loading a single cluster goes from 3.5s->.5s on a fast network link, with even more noticeable improvements on slower ones.

### CODE REVIEW

File: triage/interactive.js [hunk 1]
```diff
@@ -130,34 +131,41 @@ function rerender(maxCount) {
...
+      let graph = addElement(summary, 'div');
+      renderGraph(graph, clustered.allBuilds());
+    }
 
-  renderSubset(0, maxCount || 10);
+    renderSubset(0, maxCount || 10);
+  }
 
-  drawVisibleGraphs();
+  setTimeout(drawVisibleGraphs, 0);
```
Comment (fejta) [inline triage/interactive.js]: why timeout? Maybe comment the intent. Might also be better to encapulate this in a "drawQuickly" or something method which takes the drawVisibleGraphs function

File: triage/interactive.js [hunk 2]
```diff
@@ -130,34 +131,41 @@ function rerender(maxCount) {
...
+// Render just the cluster with the given key.
+// Show an error message if no live cluster with that id is found.
+function renderOnly(keyId) {
   var el = null;
+  rerender(0);
+
+  var top = document.getElementById('clusters');
+  top.removeChildren();
+
+  addElement(top, 'h3', null, [createElement('a', {href: ''}, 'View all clusters')]);
```
Comment (fejta) [inline triage/interactive.js]: Nice

File: triage/interactive.js [hunk 3]
```diff
@@ -130,34 +131,41 @@ function rerender(maxCount) {
...
+
   if (!clustered.byId[keyId]) {
+    var summary = document.getElementById('summary');
+    summary.innerText = `Cluster ${keyId} not found in the last week of data.`
```
Comment (fejta) [inline triage/interactive.js]: Also very nice! this still shows a view all clusters link right? I think so

File: triage/interactive.js [hunk 4]
```diff
@@ -222,34 +230,59 @@ function get(uri, callback, onprogress) {
...
+
+// One-time initialization of the whole page.
+function load() {
+  setOptionsFromURL();
+
+  var clusterId = null;
+  if (/^#[a-f0-9]{20}$/.test(window.location.hash)) {
+    clusterId = window.location.hash.slice(1);
+    // Hide options for now.
+    document.getElementById('options').style.display = 'none';
```
Comment (fejta) [inline triage/interactive.js]: Is this an in progress feature or something?

**Comment by fejta** [comment]
/lgtm
Modulo comments

File: triage/interactive.js (hunk 3 above)
Comment (rmmh) [inline triage/interactive.js]: Yes.

File: triage/interactive.js (hunk 1 above)
Comment (rmmh) [inline triage/interactive.js]: Commented with more details on what it's doing.

**Pull request merged**